build-config

```diff
@@ -28,17 +28,17 @@ else
   rm sed.script
 fi
 
-echo "Generating pre deployment file from template"		
```

Comment:
seem there's no change of this code snippet, pleace check the unnecessary spaces at the end of line